python/src/opendp/extras/numpy/_make_np_count/__init__.py

```diff
@@ -13,6 +13,10 @@ def make_np_count(input_domain: Domain, input_metric: Metric) -> Transformation:
     :param input_metric: instance of `symmetric_distance()`
     """
     dp.assert_features("contrib")
+
+    if not str(input_domain).startswith("NPArray2Domain"):
+        raise ValueError("input_domain must be NPArray2Domain")  # pragma: no cover
```

Comment:
With domain objects having more accessible structure now, can we get away from looking for substrings?

Reply:
Resolved in follow-up PR